### PR TITLE
port tgui changelog

### PR DESCRIPTION
**Pull request by ZephyrTFA**
yes

### CODE REVIEW

File: tgui/packages/tgui/interfaces/Changelog.js [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+            here
+          </a>
+          .
+        </p>
+        <p>
+          {'You can also join our discord '}
+          <a href="https://tgstation13.org/phpBB/viewforum.php?f=60">here</a>.
+        </p>
+        {dateDropdown}
+      </Section>
```
Comment (MarkSuckerberg) [inline tgui/packages/tgui/interfaces/Changelog.js]: :yamero:

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Review by MarkSuckerberg — APPROVED** [review]
Seems good, just fix the regex file and I'll merge it

**Comment by ZephyrTFA** [comment]
> Seems good, just fix the regex file and I'll merge it

Master problem

File: .github/workflows/compile_changelogs.yml [hunk 2]
```diff
@@ -2,55 +2,51 @@ name: Compile changelogs
...
           git pull origin master
+          git add html/changelogs
           git commit -m "Automatic changelog compile [ci skip]" -a || true
       - name: "Push"
-        if: steps.value_holder.outputs.CL_ENABLED
+        if: steps.value_holder.outputs.ACTIONS_ENABLED
         uses: ad-m/github-push-action@master
         with:
-          github_token: ${{ secrets.CHANGELOG_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (MarkSuckerberg) [inline .github/workflows/compile_changelogs.yml]: pretty sure this'll make it not work unless I generate a new PAT from the bot account which I can do but that takes effort

Comment (ZephyrTFA) [reply .github/workflows/compile_changelogs.yml]:
```suggestion
          github_token: ${{ secrets.CHANGELOG_TOKEN }}
```

File: .github/workflows/compile_changelogs.yml [hunk 3]
```diff
@@ -2,55 +2,51 @@ name: Compile changelogs
...
   compile:
     name: "Compile changelogs"
     runs-on: ubuntu-20.04
     steps:
-      - name: "Check for CHANGELOG_ENABLER secret and pass true to output if it exists to be checked by later steps"
+      - name: "Check for ACTION_ENABLER secret and pass true to output if it exists to be checked by later steps"
         id: value_holder
         env:
-          CHANGELOG_ENABLER: ${{ secrets.CHANGELOG_ENABLER }}
+          ENABLER_SECRET: ${{ secrets.ACTION_ENABLER }}
```
Comment (ZephyrTFA) [inline .github/workflows/compile_changelogs.yml]:
```suggestion
          ENABLER_SECRET: ${{ secrets.CHANGELOG_ENABLER }}
```

**Comment by MarkSuckerberg** [comment]
LGTM

**Comment by MarkSuckerberg** [comment]
>doesn't give it a changelog to test

crying that I didn't notice that in time

**Pull request merged**